src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -49,6 +60,7 @@ def __init__(
         file_path: FilePathType,
         config_file_path: FilePathType,
         subject_name: str = "ind1",
+        timestamps: Optional[Union[List, np.ndarray]] = None,
         verbose: bool = True,
     ):
```

Comment:
Since the big `TemporalAlignmentInterface` revamp, we've been trying to move all `timestamps` references outside of `__init__` and into the new workflow, which is to set and get them using these methods: https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/basetemporalalignmentinterface.py#L11-L56

What I would recommend here is to 

a) remove `timestamps` from `__init__`, BUT go ahead and set/reserve a `._timestamps` attribute that by default is `None` and uses the existing behavior (to use DLC timestamps) during write procedure

b) allow setting of `._timestamps` to a user defined vector using the alignment methods, and if `._timetamps` is not `None` during write procedure then use it instead of DLC timestamps (just as you have injected it into the `write_subject_to_nwb` function, which is perfectly fine)

@tuanpham96 What do you think?

Reply:
Yep, you got it!